akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -620,7 +627,7 @@ final class ClusterReceptionist(pubSubMediator: ActorRef, settings: ClusterRecep
       pubSubMediator.tell(msg, tunnel)
 
     case Heartbeat ⇒
-      log.debug("Heartbeat from client [{}]", sender().path)
+      if (verboseHeartbeatLogging) log.debug("Heartbeat from client [{}]", sender().path)
```

Comment:
In addition to this we should silence the real Cluster heartbeats also. The debug logging in this file: https://github.com/akka/akka/blob/master/akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala

Therefore the configuration property should be located in akka-cluster/reference.conf, but I think we can use the same setting for the ClusterClient.